src/_gettsim/transfers/rente.py

```diff
@@ -811,20 +962,27 @@ def ges_rente_wartezeit_5(
         See basic input variable :ref:`m_freiw_beitrag <m_freiw_beitrag>`.
     m_ersatzzeit
         See basic input variable :ref:`m_ersatzzeit <m_ersatzzeit>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
 
     Returns
     -------
-    Wartezeit in years.
+    Bool of fulfilling Wartezeit von 5 Jahren.
 
     """
-    out = (m_pflichtbeitrag + m_freiw_beitrag + m_ersatzzeit) / 12
+    out = (m_pflichtbeitrag + m_freiw_beitrag + m_ersatzzeit) / 12 >= ges_rente_params[
```

Comment:
```suggestion
    j_wartezeit =  (m_pflichtbeitrag + m_freiw_beitrag + m_ersatzzeit) / 12
    out = j_wartezeit >= ges_rente_params[
```

Same below -- the line got very long to parse now (apologies for being annoying)

Reply:
changed this